src/index.css

```diff
@@ -124,8 +124,36 @@ button {
 .board.complete {
   filter: contrast(0.5) brightness(0.5);
 }
-.game.hide-completed-boards .board.complete {
-  display: none;
+
+.game.hide-completed-boards:not(.animate-hiding) .board.complete {
+  opacity: 0;
+  visibility: hidden;
+  display: block;
+  position: absolute;
+  left: -999em;
+}
+
+.game.hide-completed-boards.animate-hiding .board.complete {
+  animation-duration: 600ms;
+  animation-name: hide;
+  animation-fill-mode: both;
+}
+
+@keyframes hide {
+  0% {
+    opacity: 1
+  }
+
+  99% {
+    opacity: 0;
```

Comment:
There needs to be a
```css
    position: initial;
    left: initial;
    display: initial;
    visibility: initial;
```
here (which I'll just add myself after merging PR), as otherwise these properties become animated and the board actually disappears 50% through the animation because `position: absolute` changes value halfway through the animation

Reply:
That explains why I couldn't get the animation to be readable at 500ms, can't believe I overlooked that